calico_upgrade/pkg/commands/constants/help.go

```diff
@@ -27,20 +27,20 @@ environments are all prefixed with "V1_"
 	ReportHelp = `
     ` + FileConvertedNames + `
       This contains a mapping between the v1 resource name and the v3 resource
-      name.  This will contain an entry for every v1 resource that was
+      name. This will contain an entry for every v1 resource that was
       migrated.
 
     ` + FileNameClashes + `
       This contains a list of resources that after conversion to v3 have
-      names that are identical to other converted resources.  This may occur
+      names that are identical to other converted resources. This may occur
```

Comment:
I think this could be improved.  Perhaps something like"

This may occur because name formats in Calico v3 are in some cases more restrictive that previous versions.  When the  conversion code modifies a name so that it adheres to the v3 format, an additional short qualifying suffix is added to the name.  In rare circumstances the converted name (with the additional qualifier) may clash with other instances of the same resource type.